docs/conf.py

```diff
@@ -441,7 +441,7 @@ def source_replace(app, docname, source):
 # Dict of replacements.
 source_replacements = {
     "{PLONE_BACKEND_MINOR_VERSION}": "6.0",
-    "{PLONE_BACKEND_PATCH_VERSION}": "6.0.13",
+    "{PLONE_BACKEND_PATCH_VERSION}": "4",
```

Comment:
This line gets rendered in the code block at https://plone6--1823.org.readthedocs.build/conceptual-guides/package-management.html#manage-backend-python-packages, which becomes this URL https://dist.plone.org/release/4/constraints.txt. Can you verify?

```suggestion
    "{PLONE_BACKEND_PATCH_VERSION}": "6.0.14",
```

Reply:
Done in b9056cf2cb3d3b906d01f8155ab095e378f9555b

Reply:
Oops, it was late at night when I did this, and I did not catch that I replaced the whole version number instead of only the last digit... Your change is good.